batch/test-locally.sh

```diff
@@ -39,7 +39,8 @@ fi
 export JOBS_TABLE=jobs-$(../generate-uid.sh)
 export JOBS_PARENTS_TABLE=jobs-parents-$(../generate-uid.sh)
 export BATCH_TABLE=batch-$(../generate-uid.sh)
-tables=($JOBS_TABLE $JOBS_PARENTS_TABLE $BATCH_TABLE)
+export BATCH_ATTRIBUTES_TABLE=batch-attributes-$(../generate-uid.sh)
```

Comment:
I don't think this script is used anymore. We should clean up the test-locally infrastructure in another PR.